packages/language-server/src/codeActionProvider.ts

```diff
@@ -258,6 +258,32 @@ export function quickFix(
         },
       })
     }
+    if (diag.severity === DiagnosticSeverity.Error &&
+    diag.message.includes('It does not start with any known Prisma schema keyword.')) {
+      let diagText = textDocument.getText(diag.range).split(/\s/)
+      const spellingSuggestion = getSpellingSuggestions(
+        diagText[0],
+        ["model", "enum", "datasource", "generator"],
```

Comment:
To make this somehow future proof. Is it possible to get all the keyword names from a binary call as opposed to hardcoding it? If you agree, please feel free to create a related issue.

Reply:
Not currently, no. But I'll create an issue 👍

Reply:
It's not, but soon the available keyword names will be inside the error message, so I can then extract the possible keywords from there.